### PR TITLE
Add new headers + update AdLog Proto

### PR DESCRIPTION
**Pull request by tdaelemans**
@Vicas @LoganWax -- Please confirm I've captured the AdLog changes, should be a quick PR.

### CODE REVIEW

File: beeswax/logs/ad_log.proto [hunk 1]
```diff
@@ -14,6 +14,7 @@ syntax = "proto2";
 package logs;
 
 option java_package = "com.beeswax.logs";
+option cc_enable_arenas = true;
```
Comment (Vicas) [inline beeswax/logs/ad_log.proto]: What does this line do?

File: beeswax/logs/ad_log.proto [hunk 2]
```diff
@@ -212,7 +214,7 @@ message ConversionLogMessage {
   optional bool is_gdpr = 10 [default = false];
 }
 
-// Next Tag: 7
```
Comment (LoganWax) [inline beeswax/logs/ad_log.proto]: Where's 7?

File: beeswax/logs/ad_log.proto [hunk 3]
```diff
@@ -212,7 +214,7 @@ message ConversionLogMessage {
...
+// Next Tag: 8
```
Comment (Vicas) [inline beeswax/logs/ad_log.proto]: This tag counter should only be updated when we add a new field to the AdLogMessage, so it should probably stay at 7

File: beeswax/logs/ad_log.proto (hunk 1 above)
Comment (LoganWax) [inline beeswax/logs/ad_log.proto]: Why are we adding this?

File: beeswax/logs/ad_log.proto (hunk 3 above)
Comment (tdaelemans) [reply beeswax/logs/ad_log.proto]: It was updated to 8 in our repo, so I updated here to reflect, but I don't see a 7 there.

File: beeswax/logs/ad_log.proto (hunk 1 above)
Comment (tdaelemans) [reply beeswax/logs/ad_log.proto]: It's on the internal version of the file.

File: beeswax/logs/ad_log.proto (hunk 2 above)
Comment (tdaelemans) [reply beeswax/logs/ad_log.proto]: See above

File: beeswax/logs/ad_log.proto (hunk 1 above)
Comment (LoganWax) [reply beeswax/logs/ad_log.proto]: This seems to be a part of the code generation specs. https://developers.google.com/protocol-buffers/docs/reference/arenas I'm not sure why we have it internally or if we want to have this be the default for external customers.

Comment (LoganWax) [reply beeswax/logs/ad_log.proto]: Could ask Dan Thompson, he seems to be the originator

File: beeswax/logs/ad_log.proto (hunk 3 above)
Comment (LoganWax) [reply beeswax/logs/ad_log.proto]: This seems to have been updated by Gil to include a field related to vast errors, but the field itself doesn't seem to have been added.

**Comment by tdaelemans** [comment]
yeah, i talked to him. it was a mistake.

**Pull request merged**